src/workspaces/common/state/useWorkspace.ts

```diff
@@ -96,11 +96,6 @@ export const useWorkspace = (namespace, name): WorkspaceDetails => {
 
   const checkGooglePermissions = async (workspace, times) => {
     try {
-      // Because checkBucketReadAccess can succeed and subsequent calls to get the bucket location or storage
-      // cost estimate may fail (due to caching of previous failure results), do not consider permissions
-      // to be done syncing until all the methods that we know will be called quickly in succession succeed.
-      // This is not guaranteed to eliminate the issue, but it improves the odds.
-      await Workspaces(signal).workspace(namespace, name).checkBucketReadAccess();
```

Comment:
I took this out because `checkBucketReadAccess` has no option to pass a user project for billing. The backend API should probably be updated at some point so that it does, but in the meantime it is not a good indicator for whether permissions are done syncing because it fails to deal with RP workspaces.

Reply:
We should not remove this check, we should fix `checkBucketReadAccess`. It is not so accurately named anymore, it does more than checking bucket read access. It also check access to the google project but more importantly, it kicks off the FastPass process when some permissions are missing.